clients/drcachesim/common/trace_entry.h

```diff
@@ -982,23 +997,31 @@ typedef struct _syscall_pt_entry_t syscall_pt_entry_t;
         (SYSCALL_METADATA_ENTRY_NUM * sizeof(syscall_pt_entry_t))
 
 typedef enum {
-    /* Index of a syscall PT entry of type SYSCALL_PT_ENTRY_TYPE_PID in the PDB header. */
+    /**
```

Comment:
Are these really public enums?  Aren't these internal to the PT code?  Why do they have Doxygen comments?

Reply:
These aren't public enums. I convert them into the internal comments format.